lib/checks/color/color-contrast.json

```diff
@@ -21,7 +21,11 @@
     "impact": "serious",
     "messages": {
       "pass": "Element has sufficient color contrast of ${data.contrastRatio}",
-      "fail": "Element has insufficient color contrast of ${data.contrastRatio} (foreground color: ${data.fgColor}, background color: ${data.bgColor}, font size: ${data.fontSize}, font weight: ${data.fontWeight}). Expected contrast ratio of ${data.expectedContrastRatio}",
+      "fail": {
+        "fgOnBgColor": "Element has insufficient color contrast of ${data.contrastRatio} (foreground color: ${data.fgColor}, background color: ${data.bgColor}, font size: ${data.fontSize}, font weight: ${data.fontWeight}). Expected contrast ratio of ${data.expectedContrastRatio}",
```

Comment:
Any time there's an object with multiple choices for the message, one should always be labeled `default`. In this case, the `fgOnBgColor` should be `default` and then you don't need to assign it a value for the message text as it can be assumed this case when `null`.